ngraph/src/ngraph/op/constant.cpp

```diff
@@ -311,11 +311,20 @@ op::Constant::Constant(const element::Type& type, const Shape& shape, const void
     : Constant(type, shape)
 {
     size_t size = ceil(shape_size(m_shape) * m_element_type.bitwidth() / 8.f);
+
     std::memcpy(get_data_ptr_nc(), data, size);
     constructor_validate_and_infer_types();
     m_all_elements_bitwise_identical = are_all_data_elements_bitwise_identical();
 }
 
+op::Constant::Constant(const element::Type& type, const Shape& shape, char* data)
```

Comment:
This ctor looks very close to `Constant(const element::Type& type, const Shape& shape, const void* data);` but has completely different behavior. This might introduce hard-to-debug issues, when one of them are called instead of other unexpectedly. I would recommend to make it more explicit that this ctor just wraps existing memory without copy. For example add empty tag struct `struct import_data_tag {};` and cror with extra parameter of this type, which will be used just as indicator.